boxoffice/models/payment.py

```diff
@@ -167,3 +169,72 @@ def calculate_weekly_refunds(item_collection_ids, user_tz, year):
         ordered_week_refunds[int(week_refund.sales_week)] = week_refund.sum
 
     return ordered_week_refunds
+
+def get_settled_transactions(date_range):
+    settled_transactions = get_settlements(date_range)
+    headers = ['settlement_id', 'transaction_type', 'order_id', 'payment_id', 'refund_id', 'item_collection', 'description', 'base_amount', 'discounted_amount', 'final_amount', 'order_paid_amount', 'transaction_date', 'settled_at', 'razorpay_fees', 'order_amount', 'credit', 'debit', 'receivable_amount', 'settlement_amount', 'buyer_fullname']
+    rows = []
+    external_transaction_msg = u"Transaction external to Boxoffice. Credited directly to Razorpay?"
+
+    for settled_transaction in settled_transactions:
+        if settled_transaction['type'] == 'settlement':
+          rows.append({
+              'settlement_id': settled_transaction['entity_id'],
+              'settlement_amount': settled_transaction['amount'],
+              'settled_at': settled_transaction['settled_at']
+
+          })
+        elif settled_transaction['type'] == 'payment':
+            payment = OnlinePayment.query.filter_by(pg_paymentid=settled_transaction['entity_id']).one_or_none()
+            if payment:
+                order = payment.order
+                rows.append({
+                    'settlement_id': settled_transaction['settlement_id'],
+                    'transaction_type': settled_transaction['type'],
+                    'order_id': order.id,
+                    'payment_id': settled_transaction['entity_id'],
+                    'razorpay_fees': settled_transaction['fee'],
+                    'transaction_date': localize_timezone(order.paid_at, 'Asia/Kolkata'),
```

Comment:
Does Razorpay do IST as a documented feature of their API? Link to docs will help.